# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-text/white-space/text-wrap-balance-line-clamp-001.html.ini
# Commit: eb0b125d3214
# Full Hash: eb0b125d3214a04c293cbe2f21184e85ae00302a
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2023-09-28 21:51:27
# Regressor Bug: 1731541
# File Overlap Count: 1
# Description:
#   Bug 1731541 - Implement text-wrap: balance for nsBlockFrame reflow. r=emilio
#   
#   A simple form of balance for short blocks, implemented by incrementally reducing
#   the effective inline-size used during line-breaking, up to the point where an
#   extra line would be created.
# ==============================================================================

diff -r af42a433722a -r eb0b125d3214 testing/web-platform/meta/css/css-text/white-space/text-wrap-balance-line-clamp-001.html.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/meta/css/css-text/white-space/text-wrap-balance-line-clamp-001.html.ini	Thu Sep 28 17:28:19 2023 +0000
@@ -0,0 +1,2 @@
+[text-wrap-balance-line-clamp-001.html]
+  expected: FAIL